<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPrintReservations</class>
 <widget class="QDialog" name="DialogPrintReservations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Imprimir lista de clientes</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Título de la lista</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editTitle">
       <property name="text">
        <string>Lista de clientes</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxFilter">
     <property name="text">
      <string>Filtrar la lista para imprimir solamente los
clientes que están entre estas dos fechas:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutDate" stretch="1,0,1">
     <item>
      <widget class="QDateEdit" name="dateIni">
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string> y </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEnd">
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelInfo">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogPrintReservations</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogPrintReservations</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateIni</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>DialogPrintReservations</receiver>
   <slot>updateFilteredList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateEnd</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>DialogPrintReservations</receiver>
   <slot>updateFilteredList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxFilter</sender>
   <signal>clicked()</signal>
   <receiver>DialogPrintReservations</receiver>
   <slot>updateInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateFilteredList()</slot>
  <slot>updateInfo()</slot>
 </slots>
</ui>
